src/test/java/com/epam/reportportal/test/TestUtils.java

```diff
@@ -0,0 +1,129 @@
+package com.epam.reportportal.test;
```

Comment:
Header.
Maybe you can configure auto header for new files.

Reply:
Added.